### ISSUE TITLE
download of SAFE product not working

### ISSUE THREAD

**Issue by CatherineTaelman** [CryosphereVirtualLab/public-notebooks#14]
Hi, 
I'm trying to download the SAFE product of Sentinel data through the CVL data portal, which redirects me to 'colhub-archive.met.no' and then asks for username and password. I am trying to login there with my account of Copernicus Scihub, but it doesn't work. Do I need another profile to access this?

**Comment by steingod** [comment]
Yes, you need to register in that portal. This is a third party software supported by ESA that doesn't have Single Sign On support. However we are synchronising usernames and passwords between the real time and archive versions of this portal software, i.e. colhub.met.no and colhub-archive.met.no, but that is a dedicated set up at MET.

**Comment by CatherineTaelman** [comment]
Okay I see, thank you. I've created an account on colhub.met.no, as there is no sign-up option on colhub-archive.met.no, but access to colhub-archive is still not working.  Is this expected due to the synchronization between both platforms that takes some time?

**Comment by steingod** [comment]
In a normal situation not, but I will check tomorrow as we have had some issues following the most recent upgrade of software to DHUS v3.

**Comment by CatherineTaelman** [comment]
Thank you!

**Comment by steingod** [comment]
Sorry for the delay. I learned that there is an issue on synchronisation of accounts between the systems. Our tech staff is working on this now.

**Comment by steingod** [comment]
This issue is scheduled to be fixed December 27th when there will be a service window 10:00-14:00 CET to address this issue so hopefully everything should work after Christmas.

**Comment by CatherineTaelman** [comment]
ok, I'll test again after Christmas:)

**Comment by CatherineTaelman** [comment]
I have tried again to access the S1 SAFE folders through CVL, but it is still not working. It seems that my account has not been synchronized to colhub-archive yet, because I can log into colhub.met.no, but not to colhub-archive.met.no.

**Comment by steingod** [comment]
The most recent information I have is that is now an open issue with the provider of the ESA software used (DHuS). This issue has been reported by MET and STFC so far and relates to newer versions of the software. .I am sorry for the inconvenience and will keep you posted.

**Comment by CatherineTaelman** [comment]
okay, thank you for the update. I have found a workaround where I access the SAFE folders through the Copernicus Scihub API, so it is no problem to complete my project

**Comment by steingod** [comment]
And you can now also ask for an account in the archive service until we have the updated software available.
